tools/benchmark.sh

```diff
@@ -48,13 +48,47 @@ function display_usage() {
   echo -e "\tVALUE_SIZE\t\t\tThe size of the values to use in the benchmark (default: 400 bytes)"
   echo -e "\tBLOCK_SIZE\t\t\tThe size of the database blocks in the benchmark (default: 8 KB)"
   echo -e "\tDB_BENCH_NO_SYNC\t\tDisable fsync on the WAL"
+  echo -e "\tNUMACTL\t\tWhen defined use numactl --interleave=all"
   echo -e "\tNUM_THREADS\t\t\tThe number of threads to use (default: 64)"
   echo -e "\tMB_WRITE_PER_SEC"
   echo -e "\tNUM_NEXTS_PER_SEEK\t\t(default: 10)"
   echo -e "\tCACHE_SIZE\t\t\t(default: 16GB)"
   echo -e "\tCOMPRESSION_MAX_DICT_BYTES"
   echo -e "\tCOMPRESSION_TYPE\t\t(default: zstd)"
-  echo -e "\tDURATION"
+  echo -e "\tBOTTOMMOST_COMPRESSION\t\t(default: none)"
+  echo -e "\tMIN_LEVEL_TO_COMPRESS\t\tValue for min_level_to_compress for Leveled"
+  echo -e "\tCOMPRESSION_SIZE_PERCENT\t\tValue for compression_size_percent for Universal"
```

Comment:
I don't think I need to keep them current with the defaults in db_bench. Maybe for some, but changing the defaults in a new release can make it harder to compare new version with versions 4, 5 and 6 when the older versions use a different value for the option and the difference is significant.

I don't think there is a great solution. One alternative is to have almost no options here and let the caller pass in everything but that just moves the problem to the caller. The current approach works for me.

Reply:
I agree that keeping up with db_bench is a futile exercise as it is too much of a moving target and there are too many options.  Having said that, I am concerned that printing some of the options here (rather than having db_bench print them) might lead to a false sense of security that what you are passing in is really the option being used.  

My suggestion was to add something to db_bench to have it do something akin to Options::Dump to std::out.  That way you know that what you passed in is having the affect you are expecting.

Reply:
You asked and I declined. I don't get the point of asking again especially when these round trips take weeks.